com.gitlab.Murmele.UDPLogger.yaml

```diff
@@ -0,0 +1,86 @@
+app-id: com.gitlab.Murmele.UDPLogger
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: UDPLogger
+rename-desktop-file: UDPLogger.desktop
+rename-appdata-file: UDPLogger.appdata.xml
+rename-icon: UDPLogger
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --filesystem=home
+- --share=network
+
+modules:
+# netcdf requires that hdf5 is build with zlib
+- name: zlib
+  buildsystem: cmake-ninja
+  builddir: true
+  sources:
+  - type: git
+    url: https://github.com/madler/zlib.git
+    tag: v1.2.12
+
+# needed by netcdf
+# z lib support needed by netcdf
+- name: hdf5
+# TODO: switch to cmake again
+  buildsystem: simple # cmake-ninja
+  builddir: true
+  build-options:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DHDF5_ENABLE_Z_LIB_SUPPORT:BOOL=ON
+  build-commands:
+    - mkdir build
+    - cmake -DHDF5_ENABLE_Z_LIB_SUPPORT:BOOL=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/app -Bbuild -G Ninja .
+    - ninja -C build
+    - ninja -C build install
```

Comment:
@hfiguiere for some reason HDF5_ENABLE_Z_LIB_SUPPORT is not applied when using cmake-ninja and without this section. Do you know why?

Reply:
I can't guess from a matformed manifest. Sorry..

Reply:
fixed